bgpd/bgp_updgrp.c

```diff
@@ -1826,9 +1833,9 @@ void peer_af_announce_route(struct peer_af *paf, int combine)
 	 */
 	if (!combine || !all_pending) {
 		update_subgroup_split_peer(paf, NULL);
-		if (!paf->subgroup)
-			return;
+		subgrp = paf->subgroup;
 
+		assert(subgrp && subgrp->update_group);
```

Comment:
Let's update the commit subject line and comments.  It is implying that this is a fix where in reality we are just addding some additional code to allow us to find the crash as it is happening.

Reply:
update commit message